build/stf-collect-logs/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: "Get builds"
+  ansible.builtin.shell:
+    cmd: |
+      echo "*** [INFO] Showing oc get builds" > {{ logfile_dir }}/post_oc_get_builds.log 2>&1
```

Comment:
Immediately, I see that I didn't add a logfile_dir, and that should actually have broken something.